src/sound/Sound.js

```diff
@@ -1233,7 +1240,7 @@ Object.defineProperty(Phaser.Sound.prototype, "volume", {
         }
         else if (this.usingAudioTag && this._sound)
         {
-            this._sound.volume = value;
+            this._sound.volume = this._globalVolume * value;
```

Comment:
Do you think this could instead be

    this._sound.volume = this.game.sound.volume * value;

and remove `this._globalVolume`, etc.?

Your choice of course. 😃